public/app/core/utils/fetch.ts

```diff
@@ -7,19 +7,20 @@ export const parseInitFromOptions = (options: BackendSrvRequest): RequestInit =>
   const isAppJson = isContentTypeApplicationJson(headers);
   const body = parseBody(options, isAppJson);
 
-  if (options?.withCredentials) {
+  if (options?.credentials) {
     return {
       method,
       headers,
       body,
-      credentials: 'include',
+      credentials: options.credentials,
     };
```

Comment:
This needs to support both options.credentials (a string) and options.withCredentials and withCredentials should have a deprecated notice in our api.